### PR TITLE
The branch were we filter out the generic sport values

### LINKED ISSUE #4977: Lots of sports_centre values are suspiciously generic
- Tennis
- Tennishalle
- Schulsporthalle
- Centro Sportivo?
- Others?

### PR DESCRIPTION
**Pull request by Identitaet**
I've gotten almost all of the values out but some of the nice regex things I want to do don't seem to work

closes #4977

### CODE REVIEW

**Comment by bhousel** [comment]
> I've gotten almost all of the values out but some of the nice regex things I want to do don't seem to work

Looks good to me, you can merge it if you want 👍 
What are the nice regex things not working?  Maybe I can help!

**Review by bhousel — APPROVED** [review]

File: data/brands/leisure/sports_centre.json [hunk 1]
```diff
@@ -169,116 +103,6 @@
         "name": "Pabellón Municipal de Deportes"
```
[the remark this answers is not in the document]
Comment (bhousel) [reply data/brands/leisure/sports_centre.json:104]: For these regular expression patterns, it won't match an accented ó unless you specifically tell it to.. 

I'd do it like this:
`^(complejo|pabell[oó]n)( municipal)? de deportes$`

- square brackets are called _character classes_ - anything in there will match (a single 'o' or 'ó')
- parenthesis make groups, and `|` in a group means "or" - ("complejo" or "pabellon"/"pabellón")
- `^` matches the beginning, `$` matches the end
- `?`,`*`,`+` are called _quantifiers_
  - `?` match previous thing 0 or 1 time
  - `*` match previous thing 0 or more times
  - `+` match previous thing 1 or more times

There is lots more to know, but I try to keep them simple.. Regular expressions are really powerful.
Here is a cheatsheet:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Cheatsheet

[the remark this answers is not in the document]
Comment (Identitaet) [reply data/brands/leisure/sports_centre.json]: Thanky you
Works now

**Comment by Identitaet** [comment]
now they are some more in there in kyrilic and arabic script but I don't know enough about the languages to filter them out

**Pull request merged**